java/arcs/android/util/AndroidLog.kt

```diff
@@ -13,22 +13,25 @@ package arcs.android.util
 
 import arcs.core.util.Log
 
+private val TAG = "Arcs"
+
 /** Initializes [Log] for tests on the JVM. */
-fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel("Arcs")) {
+fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel(TAG)) {
     Log.level = level
     Log.writer = { lvl, renderedMessage, throwable ->
         when (lvl) {
-            Log.Level.Debug -> android.util.Log.d("Arcs", renderedMessage, throwable)
-            Log.Level.Info -> android.util.Log.i("Arcs", renderedMessage, throwable)
-            Log.Level.Warning -> android.util.Log.w("Arcs", renderedMessage, throwable)
-            Log.Level.Error -> android.util.Log.e("Arcs", renderedMessage, throwable)
-            Log.Level.Wtf -> android.util.Log.wtf("Arcs", renderedMessage, throwable)
+            Log.Level.Debug -> android.util.Log.d(TAG, renderedMessage, throwable)
```

Comment:
Add this for the new verbose level?
Log.Level.Verbose -> android.util.Log.v(TAG, renderedMessage, throwable)

Reply:
Ah, thanks for catching! I went straight into a meeting and missed adding this line before sending.